portfolio/src/main/webapp/script.js

```diff
@@ -68,3 +68,19 @@ function createListElement(text) {
   liElement.textContent = text;
   return liElement;
 }
+
+/**
+ * Fetches a list of comments from the server and adds it to the DOM.
+ */
+async function getComments() {
+    const response = await fetch('/addcomment');
+    const json = await response.json();
+
+    const ulElement = document.getElementById('comments-list');
+    for (comment of json) {
+        ulElement.appendChild(createListElement(comment));
+    }
+    if (ulElement.textContent.trim() === '') {
```

Comment:
Could we use the `json` or `response` to check if there are no comments yet? this is a more reliable way of making this check.

Since these operations are asynchronous, and the response could take a bit to get back to us, checking for the textContent on the document (synchronous), could happen first. This would result in having "No comments yet." show up, followed by the comments getting populated.